refactor: migrate addon package to TypeScript (#116)
qonto/ember-prismic-dom: README.md

<Prismic::Dom @nodes={{@myPrismicDoc.data.myRichText}} />
```

## TypeScript usage

> The `Prismic::Dom` component has proper [Glint](https://github.com/typed-ember/glint) types, which allow you to get strict type checking in your templates when using TypeScript.

Reviewer: ```suggestion
Each component has proper [Glint](https://github.com/typed-ember/glint) types, which allow you to get strict type checking in your templates when using TypeScript.
```

I think in that case we shouldn't specify **only** the `Prismic::Dom` component. Because we are exporting the 4 components of the package. Each one is typed. And our registry references each one. So when extending the default registry with the `PrismicDomRegistry`, we should normally be able to benefit from strict type checking in templates for each one of our components.

Author: Very good point indeed. Done! 🙌
